secrets/alibaba/alibaba_ssm.go

```diff
@@ -227,7 +227,18 @@ func (a *AlibabaSsmManager) logError(err error) {
 func (a *AlibabaSsmManager) getSdkConfig() (*openapi.Config, error) {
 	var config *openapi.Config
 
-	if a.role != "" {
+	accessKey := os.Getenv("ALICLOUD_ACCESS_KEY")
+	secretKey := os.Getenv("ALICLOUD_SECRET_KEY")
+
```

Comment:
Minor nitpick, if you like it this way
```suggestion
	var (
		config    *openapi.Config
		accessKey = os.Getenv("ALICLOUD_ACCESS_KEY")
		secretKey = os.Getenv("ALICLOUD_SECRET_KEY")
	)
```